src/main/java/com/yahoo/sketches/counting/README.md

```diff
@@ -0,0 +1,25 @@
+# Frequency counting
```

Comment:
This doesn't belong in the code tree.  Extra documentation is possible using the /resources directories, using Overview.html files, or using package-info.java files.